client/app/services/dashboard.js

```diff
@@ -210,7 +210,7 @@ function DashboardService($resource, $http, $location, currentUser) {
       }
     });
     return _.values(_.each(globalParams, (param) => {
-      param.setValue(param.getValue()); // apply global param value to all locals
+      param.setValue(param.value); // apply global param value to all locals
```

Comment:
@kravets-levko I guess this won't affect the solution for #3853 :slightly_smiling_face: